ui/shared/SimpleTooltip.jsx

```diff
@@ -25,7 +25,6 @@ export default class SimpleTooltip extends React.Component {
         <UncontrolledTooltip
           placement={placement}
           target={this.tooltipRef}
-          className="tooltip"
```

Comment:
So now the Tooltip will work with this version! I think this was accidentally left in (bootstrap class) when this component was created.